pkg/controller/astarte/reconcile/astarte_dashboard.go

```diff
@@ -159,12 +159,12 @@ func getAstarteDashboardPodSpec(deploymentName string, cr *apiv1alpha1.Astarte,
 func getAstarteDashboardConfigMapData(cr *apiv1alpha1.Astarte, dashboard apiv1alpha1.AstarteDashboardSpec) map[string]string {
 	dashboardConfig := make(map[string]interface{})
 	if dashboard.Config.RealmManagementAPIURL == "" {
-		dashboardConfig["realm_management_api_url"] = getBaseAstarteAPIURL(cr) + "/realmmanagement/v1/"
+		dashboardConfig["realm_management_api_url"] = getBaseAstarteAPIURL(cr) + "/realmmanagement/"
```

Comment:
This has to be made conditional to support 0.10.x. See for example https://github.com/astarte-platform/astarte-kubernetes-operator/blob/master/pkg/controller/astarte/reconcile/astarte_generic_api.go#L275 .

You would add a < 0.11.0-beta.3 constraint, with a TODO to remove the Beta constraint once we reach 0.11.0.